.github/workflows/nextstrain-docker-base.yml

```diff
@@ -0,0 +1,39 @@
+# Trigger rebuild in https://github.com/nextstrain/docker-base
+name: nextstrain-docker-base
+
+on:
+  release:
+    types:
+      - released
+
+  repository_dispatch:
+    types: nextstrain-docker-base
+
+  workflow_dispatch:
+
+  workflow_call:
+
+  pull_request:
```

Comment:
> How do we test that?

There's a couple options: https://github.com/nextstrain/docs.nextstrain.org/commit/aa6f75e103b83676345e3a0f7ac60291aa99e4da

I've pushed 9bd5fc5b05b439e587268f174cf1b9b5aea335a5 to test.